spec/yaml/swiftnav/sbp/bootload.yaml

```diff
@@ -22,15 +22,40 @@ include:
   - types.yaml
 definitions:
 
- - MSG_BOOTLOADER_HANDSHAKE:
+ - MSG_BOOTLOADER_HANDSHAKE_HOST:
+    id: 0x00B3
+    public: True
+    short_desc: Bootloading handshake request (host => device)
+    desc: |
+      The handshake message request from the host establishes a
+      handshake between the device bootloader and the host. The
+      response from the device is MSG_BOOTLOADER_HANDSHAKE_DEVICE.
+
+ - MSG_BOOTLOADER_HANDSHAKE_DEVICE:
     id: 0x00B0
     public: True
-    short_desc: Bootloading handshake (host <=> device)
+    short_desc: Bootloading handshake response (host <= device)
     desc: |
-      The handshake message establishes a handshake between the device
-      bootloader and the host.  The payload string contains the
-      bootloader version number, but returns an empty string for
-      earlier versions.
+      The handshake message response from the device establishes a
+      handshake between the device bootloader and the host. The
+      request from the host is MSG_BOOTLOADER_HANDSHAKE_HOST.  The
+      payload string contains the bootloader version number, but
+      returns an empty string for earlier versions.
+    fields:
+      - handshake:
+          type: array
+          fill: u8
+          desc: Version number string (not NULL terminated)
+
+ - MSG_BOOTLOADER_HANDSHAKE_OLD:
+    id: 0x00B0
+    public: False
+    short_desc: |
+      Deprecated, use MSG_BOOTLOADER_HANDSHAKE_HOST (0x00B3) and
```

Comment:
Maybe feature creep for the spec, but perhaps it might be useful to replace by:

```yaml
id: 0x00B0
public: False
deprecated: True
deprecated_in: 0.12
replaced_by:   
  - MSG_BOOTLOADER_HANDSHAKE_HOST
  - MSG_BOOTLOADER_HANDSHAKE_DEVICE
```

Also, I think the short description here might be too long for the latex columns in the datasheet. I may be wrong though.